New mirror

PardesiCat opened the pull request:
Created PR  from ikstream order 

details in

 https://github.com/BlackArch/blackarch-site/issues/175

PardesiCat committed:
New mirror 

Created PR  from ikstream order 

details in

 https://github.com/BlackArch/blackarch-site/issues/175

noraj commented on blackarch-mirrorlist:
```diff
@@ -92,6 +92,9 @@ Server = https://ftp.halifax.rwth-aachen.de/blackarch/$repo/os/$arch
 #Server = http://mirror.yandex.ru/mirrors/blackarch/$repo/os/$arch
 #Server = ftp://mirror.yandex.ru/mirrors/blackarch/$repo/os/$arch
 
+#South Korea
```
```suggestion
# South Korea
```

noraj replied:
If you can just fix the country spacing so it's like for the others. Outside of this LGTM.

PardesiCat replied:
oke did

PardesiCat committed:
New mirror 

Created PR from ikstream order + applied noraj suggestion

details in

https://github.com/BlackArch/blackarch-site/issues/175

noraj reviewed (approved):
LGTM

PardesiCat commented:
B U M P

noraj commented:
@ikstream @noptrix You guys can handle it? I have received order not to merge mirror related stuff because it need testing.
